src/main/scala/com/itechart/ny_accidents/Application.scala

```diff
@@ -1,93 +1,36 @@
 package com.itechart.ny_accidents
 
-import com.google.inject.Guice
-import com.itechart.ny_accidents.constants.Configuration
-import com.itechart.ny_accidents.constants.GeneralConstants._
-import com.itechart.ny_accidents.database.dao.PopulationStorage
+import com.itechart.ny_accidents.constants.{Configuration, Injector}
 import com.itechart.ny_accidents.database.dao.cache.MergedDataCacheDAO
 import com.itechart.ny_accidents.entity.{Accident, MergedData}
 import com.itechart.ny_accidents.parse.AccidentsParser
-import com.itechart.ny_accidents.report.Reports
+import com.itechart.ny_accidents.report.ReportsGenerator
 import com.itechart.ny_accidents.service.MergeService
-import com.itechart.ny_accidents.service.metric.{DistrictMetricService, PopulationMetricService, TimeMetricService, WeatherMetricService}
+import com.itechart.ny_accidents.service.metric.PopulationMetricService
 import com.itechart.ny_accidents.utils.FileWriterUtils
 import org.apache.spark.rdd.RDD
 import org.slf4j.LoggerFactory
 
 object Application extends App {
   lazy val logger = LoggerFactory.getLogger(getClass)
 
-  val injector = Guice.createInjector(new GuiceModule)
-  val accidentsParser = injector.getInstance(classOf[AccidentsParser])
-  val mergeService = injector.getInstance(classOf[MergeService])
-  val weatherMetricService = WeatherMetricService
-  val cacheService = injector.getInstance(classOf[MergedDataCacheDAO])
-  val populationService = injector.getInstance(classOf[PopulationMetricService])
-  val populationStorage = injector.getInstance(classOf[PopulationStorage])
+  val accidentsParser = Injector.injector.getInstance(classOf[AccidentsParser])
+  val mergeService = Injector.injector.getInstance(classOf[MergeService])
+  val cacheService = Injector.injector.getInstance(classOf[MergedDataCacheDAO])
```

Comment:
Maybe let's import Injector.injector then you will access _injector_ directly

Reply:
Yes, thank you. Rewrited